Fleece/PlatformCompat.cc

```diff
@@ -0,0 +1,48 @@
+#ifdef WIN32
```

Comment:
I left it like that because on windows no matter the compiler we want these definitions, but I just tried out clang and it defines `_MSC_VER` also...and for some reason doesn't define anything like `__clang__` or `__llvm__` that I can tell...so for now I'll just say don't try to use alternative compilers on Windows and change them both to `_MSC_VER`